Neos.Cache/Classes/Backend/RedisBackend.php

```diff
@@ -396,13 +396,13 @@ public function isFrozen(): bool
     /**
      * Sets the default lifetime for this cache backend
      *
-     * @param integer $lifetime Default lifetime of this cache backend in seconds. If NULL is specified, the default lifetime is used. 0 means unlimited lifetime.
+     * @param integer|string|null $lifetime Default lifetime of this cache backend in seconds. If NULL is specified, the default lifetime is used. 0 means unlimited lifetime.
      * @return void
      * @api
      */
-    public function setDefaultLifetime(int $lifetime): void
+    public function setDefaultLifetime($lifetime): void
     {
-        $this->defaultLifetime = $lifetime;
+        $this->defaultLifetime = $lifetime === null ? null : (int)$lifetime;
```

Comment:
How about using `is_numeric()` here, as a safeguard against stupid strings? We could throw an `InvalidArgumentException` if it's not null or numeric.

Reply:
Given that so far we didn't safeguard against anything here… and it would probably be cast to `0`, which means unlimited. Fine with me. ;)

Reply:
✅